solr/core/src/test/org/apache/solr/security/BaseTestRuleBasedAuthorizationPlugin.java

```diff
@@ -213,14 +212,14 @@ public void testBasicPermissions() {
         "userPrincipal", "tim",
         "handler", new ReplicationHandler(),
         "collectionRequests", singletonList(new CollectionRequest("mycoll")) )
-        , FORBIDDEN);
+        , STATUS_OK); // Replication requires "READ" permission, which Tim has
```

Comment:
The user "tim" has the `dev` and `admin` roles, and thus the permission `read`, `collection-admin-edit` and `schema-edit`. 

Before this PR, `/replication` was not covered by any particular permission, so replication request was denied, since the user did not either have the `all` permission. After the PR, access is allowed since he has `read`.

Reply:
Thanks for these helpful explanations.  Makes reviewing very easy.